CHANGELOG.md

```diff
@@ -58,6 +58,7 @@ Main (unreleased)
   - `query_sample`: add option to use TiDB sql parser (@cristiangreco)
   - `query_tables`: rename collector from `query_sample` to better reflect responsibility (@matthewnolf)
   - `query_sample`: add new collector that replaces previous implementation to collect more detailed sample information (@matthewnolf)
+  - component now also collects metrics on enabled `performance_schema.setup_consumers` (@fridgepoet)
```

Comment:
The PR needs a rebase I think, after which this changelog entry should be moved to the "Unreleased" section.

Reply:
Done